docs/cmd/kn_service_create.md

```diff
@@ -45,6 +48,7 @@ kn service create NAME --image IMAGE [flags]
       --annotation stringArray    Service annotation to set. name=value; you may provide this flag any number of times to set multiple annotations. To unset, specify the annotation name followed by a "-" (e.g., name-).
       --async                     Create service and don't wait for it to become ready.
       --autoscale-window string   Duration to look back for making auto-scaling decisions. The service is scaled to zero if no request was received in during that time. (eg: 10s)
+      --cluster-local             specify that the service be private; sets service's route visibility to 'cluster-local'.(--no-cluster-local will remove any 'cluster-local' route visibility)
```

Comment:
Minor thing, but I'd remove the text about routes as they don't need to know how we're doing it. So perhaps just:
```
specify that the service be private. (--no-cluster-local will make the service publicly available)
```

Reply:
same for other help texts